apisix/plugins/grpc-transcode/util.lua

```diff
@@ -51,7 +51,7 @@ local function get_from_request(name, kind)
     local request_table
     if ngx.req.get_method() == "POST" then
         if string.find(ngx.req.get_headers()["Content-Type"] or "",
-                       "application/json", true) then
+                       "application/json") then
```

Comment:
Doesn't the original code mean `string.find(str, pattern, 1, true)`?

Reply:
yes. thanks.